packages/renderer/src/lib/markdown/micromark-listener-handler.ts

```diff
@@ -76,44 +67,47 @@ export function createListener(
       }
     }
 
+    // BUTTON CHECK: 'expandable'
     // if the user clicked on the toggle of an expandable section
     if (expandable) {
       executeExpandableToggle(expandable);
       return;
     }
 
+    // BUTTON CHECK: 'generic' button click (not a command)
     // if the user clicked on a button (new way)
-    if (!command && eventTarget instanceof HTMLButtonElement) {
-      const targetId = eventTarget.id;
+    if (!command && e.target instanceof HTMLButtonElement) {
+      const targetId = e.target.id;
       executeButtonCommand(targetId).catch((err: unknown) => console.error(`Error executing command ${targetId}`, err));
       return;
     }
 
+    // BUTTON CHECK: 'command' button click (used in onboarding package.json)
     // Only check if the command exists and the target is not disabled
-    if (command && 'disabled' in eventTarget && !eventTarget.disabled) {
+    if (command) {
       // If the target is an instance of a button element, we know that we are going to execute either
       // a command or hyperlink
-      if (eventTarget instanceof HTMLButtonElement) {
-        const targetButton = eventTarget as HTMLButtonElement;
+      if (e.target instanceof HTMLButtonElement && !e.target.disabled) {
         // If the command exists and the button is not disabled, we execute the command
         // we'll also be updating the inProgressMarkdownCommandExecutionCallback so we have
         // real-time updates on the button
         inProgressMarkdownCommandExecutionCallback(command, 'starting');
-        targetButton.disabled = true;
-        if (targetButton.firstChild && targetButton.firstChild instanceof HTMLElement) {
-          targetButton.firstChild.style.display = 'inline-block';
+        e.target.disabled = true;
+        if (e.target instanceof HTMLElement && e.target.firstChild instanceof HTMLElement) {
```

Comment:
HTMLButtonElement extends HTMLElement. If e.target is an instance of HTMLButtonElement it is also an instance of HTMLElement. So thins instanceof check not needed IMO.

Reply:
Linting error when removed, I get `Property 'style' does not exist on type 'ChildNode'`, so would need to keep this in.